src/views/ArticleBody.jsx

```diff
@@ -22,23 +29,28 @@ function ArticleBody(props) {
       <div className="single__author__container">
         <div className="article__author">
           <div className="author__profile--mini">
-            <img src={article.image} alt="" />
+            <img src={profile.image} alt="author" />
             <div className="author__profile--details">
-              <h6>Alexandra</h6>
-              <span>22 hrs ago</span>
+              <h6>{`${profile.firstname} ${profile.lastname}`}</h6>
+              <span>{`${time(article)} ago`}</span>
               <span className="article-author-dot" />
-              <span>4 min read</span>
+              <span>{article.readTime}</span>
             </div>
           </div>
           <div className="bookmark__div">
-            <button onClick={bookmarkArticle} type="button">
+            <button
+              onClick={debounceFn(bookmarkArticle, 500)}
+              type="button"
+              className="btn btn-link bookmark-btn"
+              id="bookmark-btn"
+            >
               <span className={`fas fa-bookmark transition ${(bookmarked) ? 'bookmarked' : ''}`} />
             </button>
           </div>
         </div>
       </div>
-      <div className="single-body">
-        {article.body}
+      <div className="single-body" dangerouslySetInnerHTML={returnMarkup()}>
+        {/* {article.body} */}
```

Comment:
#### Nitpick
- Why the comment?

Reply:
So initially, what was supposed to happen was that the article being created was supposed to be encoded. The comment was meant to be where the encoded body would be decoded by the unescape package. 

However, the created article comes in this format from the DB (see image below. Emphasis on the body)
<img width="1125" alt="Screenshot 2019-04-24 at 8 20 33 AM" src="https://user-images.githubusercontent.com/24186167/56640304-95f5ef00-666a-11e9-9f55-10851e8f0962.png">

The above resulted in the image below. So the comment was there pending when we can resolve the issue. It has been resolved now. 
<img width="798" alt="Screenshot 2019-04-24 at 8 42 37 AM" src="https://user-images.githubusercontent.com/24186167/56654240-83d67980-6687-11e9-92ae-43c84d9656ed.png">